compiler/rustc_const_eval/src/const_eval/machine.rs

```diff
@@ -249,7 +249,7 @@ impl<'tcx> CompileTimeInterpCx<'tcx> {
             return Err(ConstEvalErrKind::Panic { msg, file, line, col }).into();
         } else if self.tcx.is_lang_item(def_id, LangItem::PanicFmt) {
             // For panic_fmt, call const_panic_fmt instead.
-            let const_def_id = self.tcx.require_lang_item(LangItem::ConstPanicFmt, None);
+            let const_def_id = self.tcx.require_lang_item(LangItem::ConstPanicFmt, self.tcx.span);
```

Comment:
That's a very imprecise span (it points at the root constant, not the current operation). `self.cur_span()` is a lot better but also more expensive. But this is only executed when a panic happens so that seems fine?

Reply:
it's a span only used for stating that a lang item is missing. In my ideal world it would be a closure, because then I could make other places also compute better spans, but it's just not worth it for a case of "your libcore is wrong"